unified-codes/apps/web/src/components/containers/explorer/ExplorerPage.tsx

```diff
@@ -24,14 +24,12 @@ const useStyles = makeStyles((theme: ITheme) =>
       backgroundColor: theme.palette.background.default,
       margin: '0 auto 0 auto',
       maxHeight: '100%',
-      maxWidth: 900,
       width: '60%',
       borderRadius: 5,
     },
     searchBarContainer: {
       backgroundColor: theme.palette.background.default,
       margin: '-20px auto 0 auto',
```

Comment:
the original design mandated a max width. I think ok to remove, though I'm concerned about the resulting very wide table possibilities. harder to read very wide tables.